talos-v1.11.yaml

```diff
@@ -201,14 +201,31 @@ navigation:
         - group: "Reference"
           folder: "talos/v1.11/reference"
           pages:
+            - "v1alpha1/config.mdx"
```

Comment:
I wanted to bring this up to the top level of the references because it's the most common and important configuration to reference. All other configuration folders were flattened into 1 page if there was only 1 configuration in that group.

<img width="326" height="445" alt="image" src="https://github.com/user-attachments/assets/b625aac9-cb9e-4909-b867-8c5e3fb99ffa" />

We could keep the groups but then someone has to expand the section before they can see the config documents in that group (IMO it's annoying)